activeadmin-mongoid.gemspec

```diff
@@ -17,10 +17,10 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.version       = ActiveAdmin::Mongoid::VERSION
   gem.license       = 'MIT'
-  
+
   gem.required_ruby_version = '>= 2.3'
 
-  gem.add_runtime_dependency 'mongoid',     ['~> 6.0']
+  gem.add_runtime_dependency 'mongoid',     '>= 6.0', '<= 7.0'
```

Comment:
I think this is only going to allow mongoid 7.0.0, but not higher versions, and I don't think that's what you want? I think what you want is

```suggestion
  gem.add_runtime_dependency 'mongoid',     '>= 6.0', '< 8.a'
```

which allows any mongoid 7 version, but disallows any moingoid 8 version, including potential prereleases:

### Before

```
irb(main):009:0> Gem::Requirement.new(">= 6.0", "<= 7.0").satisfied_by?(Gem::Version.new("7.0.0"))
=> true
irb(main):010:0> Gem::Requirement.new(">= 6.0", "<= 7.0").satisfied_by?(Gem::Version.new("7.0.1"))
=> false
```

### After

```
irb(main):003:0> Gem::Requirement.new(">= 6.0", "< 8.a").satisfied_by?(Gem::Version.new("7.0.0"))
=> true
irb(main):004:0> Gem::Requirement.new(">= 6.0", "< 8.a").satisfied_by?(Gem::Version.new("7.0.1"))
=> true
```